src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java

```diff
@@ -753,6 +749,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 		}
 	}
 
+	private void addLibraryJar(ArrayList<String> args, ArrayList<File> libraryJars, File file)
+	{
+		if (putLibraryJarsInTempDir) {
+			libraryJars.add(file);
+		} else {
+			args.add("-libraryjars");
+			args.add(fileToString(file));
+			if (libraryJarExclusion != null) {
+				args.add(libraryJarExclusion);
```

Comment:
If I understood correctly, this exclusion string should be added immediately after the filename without any spaces between them. This would seem to add it as a separate argument for the command.